### ISSUE TITLE
Support for sandbox and test environments in Salesforce destination

### ISSUE THREAD

**Issue by muscovitebob** [grouparoo/grouparoo#2278]
**Is your feature request related to a problem? Please describe.**
In order to validate Grouparoo for my use case I need to connect to a Salesforce sandbox since doing changes to my production Salesforce instance is not allowed by the organisation. At the moment there does not appear to be an option to change the Salesforce instance URL.

**Describe the solution you'd like**
Alongside login, password and token, the ability to provide a Salesforce instance URL in a Salesforce Destination.

**Describe alternatives you've considered**
Modifying the source code to uncomment the following in `connect.ts`, however this solution requires one to self host instead of using the managed Grouparoo offering, which is of interest to me.
```js
  var conn = new jsforce.Connection({
    // you can change loginUrl to connect to sandbox or prerelease env.
    // loginUrl : 'https://test.salesforce.com'
  });
```

**Comment by bleonard** [comment]
Hi @muscovitebob - thanks for the note.
That sounds like a good idea. Based on what you know, do you think it should be a checkbox and that always mean `'https://test.salesforce.com'` or are these lots of these environments and it should be a text box fields to give the whole url?

**Comment by muscovitebob** [comment]
Hey @bleonard I think the safest would indeed be a text box option. I think you can reach most sandboxes via the test sub domain but I am not completely certain that it is always the case. Thanks for your time.

**Comment by bleonard** [comment]
Cool, we will look into it. Pardot already has that option so we will follow that approach.

**Comment by bleonard** [comment]
Released in 0.6.2 and it's now on our hosted builds.
<img width="476" alt="Screen Shot 2021-09-08 at 7 44 16 PM" src="https://user-images.githubusercontent.com/36106/132613857-aad842e4-5b78-4036-a5ad-d1533ec3f328.png">
